lunatrace/bsl/backend/src/snapshot/node-package-tree.ts

```diff
@@ -255,20 +261,20 @@ async function findCurrentlyKnownDependencies(query: string, manifestIds: string
 
     log.info(`Checking ${slice.length} currently known dependencies (${i + slice.length}/${manifestIds.length})`);
 
-    const result = await db.manyOrNone<string>(query, [slice]);
+    const result = await db.manyOrNone<{ id: string }>(query, [slice]);
 
     log.info(`Added ${result.length} known dependencies (${i + slice.length}/${manifestIds.length})`);
 
     if (result) {
-      currentlyKnownIds.push(result);
+      for (const { id } of result) {
+        currentlyKnownIds.add(id);
+      }
     }
   }
 
-  const allKnownIds = currentlyKnownIds.flat(1);
+  log.info(`Found ${currentlyKnownIds.size} known dependencies`);
```

Comment:
the pattern that I prefer for logging is:

```
log.info('Found known dependencies', { count: currentlyKnownIds })
```

it is a lot easier to parse logs when the log message is consistent vs having to read a sentence

Reply:
That's a good point